packages/react-components/src/components/RTE/RichTextEditor.tsx

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import * as React from 'react';
+import { ContentEdit, Watermark } from 'roosterjs-editor-plugins';
+import { Editor } from 'roosterjs-editor-core';
+import { EditorOptions, IEditor } from 'roosterjs-editor-types';
+import {
+  Rooster,
+  createRibbonPlugin,
+  createUpdateContentPlugin,
+  Ribbon,
+  getButtons,
+  RibbonButton,
+  KnownRibbonButtonKey,
+  UpdateMode
+} from 'roosterjs-react';
+import { richTextEditorStyle } from '../styles/RichTextEditor.styles';
+
+/**
+ * Props for {@link RichTextEditor}.
+ *
+ * @beta
+ */
+export interface RichTextEditorProps {
+  content?: string;
+  onChange: (newValue?: string) => void;
+  children: React.ReactNode;
```

Comment:
`InputBoxComponent` uses a `children` prop because `onRenderSuffix` prop was used to render buttons in FluentUI `TextField` component.
 Why do we need the `children` prop here?

Reply:
In the new design, the button is horizontally aligned with the ribbon buttons within the Rich Text Editor. In future implementations, we should consider increasing the separation between these buttons and the text editor itself for better clarity and usability. For now, insert the children components here to ensure the Rich Text Editor is operational.